snorkel/labeling/__init__.py

```diff
@@ -10,4 +10,5 @@
     RandomVoter,
 )
 from .model.label_model import LabelModel  # noqa: F401
+from .model.tuner import LabelModelTuner  # noqa: F401
```

Comment:
need to remove to keep `ray` off the main import path. can revisit later, but doesn't look essential right now.